create_table.lua

```diff
@@ -15,8 +15,8 @@ w_street_2 varchar(20), \
 w_city varchar(20), \
 w_state varchar(2), \
 w_zip varchar(9), \
-w_tax varchar(7), \
-w_ytd varchar(15), \
+w_tax double, \
+w_ytd double, \
```

Comment:
I think that together with the fix we should file an issue to replace this direct but possibly incorrect solution with one that is proven to correspond to the [standard](https://tpc.org/tpc_documents_current_versions/pdf/tpc-c_v5.11.0.pdf).

I would expect the following information to be present in the issue.

1. Which doubts about correctness of the current implementation exists (you can also link [this](https://github.com/tarantool/tpcc/pull/7#discussion_r542201890) comment).
2. Why the native `decimal` tyoe is not used currently and what are the problems you see on the road to it.

Reply:
Done (see #10)

Reply:
Thank you!